appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockChart.java

```diff
@@ -35,9 +39,16 @@ protected MockChart(SimpleEditor editor, String type, ImageResource icon) {
     protected void initChart() {
         chartWidget.getOptions().setMaintainAspectRatio(false);
         chartWidget.getOptions().getTitle().setDisplay(true);
-        chartWidget.setStylePrimaryName("ode-SimpleMockComponent");
 
-        initComponent(chartWidget);
+        chartPanel = new AbsolutePanel();
+        chartPanel.setStylePrimaryName("ode-SimpleMockComponent");
+
+        rootPanel.setHeight("100%");
+
+        rootPanel.add(chartWidget);
```

Comment:
rootPanel.add should (generally) be avoided according to comments.

Maybe there is another workaround?

Reply:
For now, this solution will be used due to the fact that the Data components should be dragged onto the Chart itself.